traffic_ops/build/traffic_ops.spec

```diff
@@ -122,7 +122,7 @@ Built: %(date) by %{getenv: USER}
     # install
     if [ "$1" = "1" ]; then
       # see postinstall, the .reconfigure file triggers init().
-      /bin/touch %{PACKAGEDIR}/.reconfigure
+      #/bin/touch %{PACKAGEDIR}/.reconfigure
```

Comment:
Removed because reconfigure is now a command line option - removed line from spec file